### DISCUSSION TITLE
Should/Could we include a health endpoint in the Azure Functions project templates?

### DISCUSSION THREAD

**Discussion by stijnmoreels** [arcus-azure/arcus.templates#320] [Q&A] [answered]
**What is your question/concern?**
Could we include an endpoint that has the health status, like we did for the Web API but in the context of Azure Functions?
Seems like this is something that will also be added a lot when people create Azure Functions?
First proposed by @pim-simons

**Comment by tomkerkhove** [comment, marked as answer]
Will be done as part of https://github.com/arcus-azure/arcus.templates/issues/378
